times_reader/transforms.py

```diff
@@ -1768,8 +1772,7 @@ def has_no_wildcards(list):
                 and set(df.columns) & query_columns == {"cset_cn"}
                 and has_no_wildcards(df["cset_cn"])
             ):
```

Comment:
@olejandro do you know if this `if` condition and the `elif` directly below can be replaced by something more general? Perhaps this is something we also do to other `TFM_INS` variants?

Reply:
Basically, @siddharth-krishna I believe this part can just be dropped if we do reduction earlier.

Reply:
Btw, on generalising the code: we'll need to do this kind of reduction also for `tfm_upd` and `tfm_dins`